### PR TITLE
Added image processing

### PR DESCRIPTION
**Pull request by DavidVollendroff**
# Description
Changes necessary to affix input preprocessing which completes RC L22.1 by meeting the final unmet **Key Result**; _Image inputs lead to classification outputs_

Fixes # (converts uploaded images to a format suitable for classification)

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [X] Test A
test_image

This image was created using the training data CSV and reshaping the values back into an image file. It is a 28x28 png image in grayscale. It was was uploaded via a POST request to the API and our model successfully predicted it is the letter 'D'.

- [X] Test B
test_image_D
To test for robustness I then uploaded this 640x640 resolution, RGB,  jpeg image. The image processing function successfully transformed the input into a form usable by the classifier, and an output was obtained.

# Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] There are no merge conflicts

### CODE REVIEW

**Review by Cal-0 — APPROVED** [review]
Beautiful.

**Review by nwilliams030 — APPROVED** [review]

**Review by ivaben — APPROVED** [review]
nice

File: utilities.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# Import statements
+import tensorflow as tf
 
 
 def process_img(input_image):
-    """Placeholder function that will be eventually be used to process our uploaded images into a model-usable form."""
+    """
```
Comment (ryan-hamblin) [inline utilities.py:5]: I see you've used two different styles of commenting your code. I would suggest we standarize as a team.

File: asl.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
-from flask import Flask, request
+import tensorflow as tf
+import numpy as np
+from flask import Flask, request, jsonify
 from .models import initial_model
 from .utilities import process_img
 
-model = initial_model()  # created upon deployment instead of every call of predict
 
+model = initial_model()  # created upon deployment instead of every call of predict
```
Comment (ryan-hamblin) [inline asl.py:8]: Thank you so much for optimizing this function, as well as commenting about how it's performing.

**Pull request merged**